pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -1,136 +1,228 @@
-                                <%- include('../partials/assessments', {assessments: question.assessments}); %>
-                            </td>
-                            <% } %>
-                        </tr>
-                        <% }); %>
-                    </tbody>
-                </table>
+            <div class="col-auto">
+              <% if ((authz_data.has_course_permission_edit) && (! course.example_course) && (! locals.needToSync)) { %>
+              <form class="d-inline" name="add-question-form" method="POST">
+                <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                <button name="__action" value="add_question" class="btn btn-sm btn-light"><i class="fa fa-plus" aria-hidden="true"></i>
+                  <span>Add question</span></button>
+              </form>
+              <% } %>
             </div>
-            <script>
-                $(function() {
-                    $("#questionsTable").tablesorter({
-                        theme: "bootstrap",
-                        widthFixed: true,
-                        headerTemplate: '{content} {icon}',
-                        widgets: ["uitheme", "filter", "zebra"],
-                        widgetOptions: {
-                            zebra: ["even", "odd"],
-                            filter_reset: "#resetFilter",
-                            filter_cssFilter: "form-control",
-                            filter_functions: {
-                                3: {
-                                    <% all_tags.forEach(function(tag) { %>
-                                    "<%= escapeRegExp(tag.name) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(tag.name) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% if (locals.course_instance) { %>
-                                5: {
-                                    <% all_assessments.forEach(function(assessment) { %>
-                                    "<%= escapeRegExp(assessment.label) %>": function(e, n, f, i, $r, c, data) {
-                                        return /\b<%= escapeRegExp(assessment.label) %>\b/.test(e);
-                                    },
-                                    <% }); %>
-                                },
-                                <% } %>
-                            },
-                        },
-                    });
-                });
-            </script>
+          </div>
+        </div>
+
+        <table id="questionsTable"
+               data-data="<%= JSON.stringify(questions) %>"
+               data-classes="table table-sm table-hover table-bordered"
+               data-thead-classes="thead-light"
+               data-filter-control="true"
+               data-show-filter-control-switch="true"
```

Comment:
Do we really want this button? Do you think people want to be able to hide the filter controls?

<img width="130" alt="image" src="https://user-images.githubusercontent.com/875833/128927757-2da96665-95f9-46f9-b0f4-8d3a35a10475.png">